tests/rembert/test_modeling_tf_rembert.py

```diff
@@ -168,7 +168,55 @@ def create_and_check_model(
 
         self.parent.assertEqual(result.last_hidden_state.shape, (self.batch_size, self.seq_length, self.hidden_size))
 
-    def create_and_check_lm_head(
+    def create_and_check_causal_lm_base_model(
+        self, config, input_ids, token_type_ids, input_mask, sequence_labels, token_labels, choice_labels
+    ):
+        config.is_decoder = True
+
+        model = TFRemBertModel(config=config)
+        inputs = {"input_ids": input_ids, "attention_mask": input_mask, "token_type_ids": token_type_ids}
+        result = model(inputs)
+
+        inputs = [input_ids, input_mask]
+        result = model(inputs)
+
+        result = model(input_ids)
```

Comment:
Same question as below here - I don't understand why we overwrite `result` three times, except to check that it doesn't crash in the earlier calls?

Reply:
Yes, that's what I think too. Just to make sure it runs.

Reply:
This (several runs but only test one result) is already in the current master.
This PR just cleans thing up, and add missing tests for some tests (mostly, for `past_key_values`)